pkg/scoring/atlassian_test.go

```
@@ -0,0 +1,319 @@
+		},
+	}
+	_, _, _, ok := extractAtlassianCredentials(m)
+	assert.False(t, ok)
+}
+
+func TestAtlassianSiteAdminCondition_IsAdmin(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		switch r.URL.Path {
+		case "/rest/api/3/myself":
+			w.WriteHeader(http.StatusOK)
+			json.NewEncoder(w).Encode(map[string]interface{}{
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)